superset/databases/schemas.py

```diff
@@ -327,6 +329,7 @@ class Meta:  # pylint: disable=too-few-public-methods
         description=cache_timeout_description, allow_none=True
     )
     expose_in_sqllab = fields.Boolean(description=expose_in_sqllab_description)
+    configuration_method = EnumField(ConfigurationMethod, by_value=True)
```

Comment:
nit: add a small description here

Reply:
`  # configuration_method is used on the frontend to inform the backend whether to explode parameters or to provide only a sqlalchemy_uri
    configuration_method = EnumField(ConfigurationMethod, by_value=True)`

Does this look ok?

Reply:
We have two choices here.. let's discuss. When saving the database, the return is a sqlalchemy_uri, regardless of how the database was created. The new form is a two step form, so you basically create and then update. My current plan (pr isn't up yet, but I can paste it here for reference) is to not provide the configuration_method in the put. There are no plans as of now to allow people to switch from one to the other once they have created the database. But @yousoph and @steejay and I have been talking about this, so it could change. The other option is to return the parameters and the configuration_method value with the save db response, which would be a more flexible solution, but might take more time to implement, and as it looks now we don't need it, so it could wait for a v2.